testsuite/MDAnalysisTests/analysis/test_rms.py

```diff
@@ -128,6 +129,70 @@ def test_with_superposition_equal(self):
         rmsd_superposition = rms.rmsd(A, B, center=True, superposition=True)
         assert_almost_equal(rmsd, rmsd_superposition)
 
+class TestRMSD(object):
+    def setUp(self):
+        self.universe = MDAnalysis.Universe(PSF, DCD)
+        self.tempdir = tempdir.TempDir()
+        self.outfile = os.path.join(self.tempdir.name, 'rmsd.npy')
+
+    def tearDown(self):
+        del self.universe
+        del self.tempdir
+
+    def test_rmsd(self):
+        RMSD = MDAnalysis.analysis.rms.RMSD(self.universe, select='name CA')
+        RMSD.run(step=49)
+        correct_values = [[  0, 0, 0],
```

Comment:
store the `correct_values` in the class (a "fixture") so that you can refer to them here and in `test_mass_weighted_and_save()`.